fluid/alignment_panel.fl

```diff
@@ -47,11 +50,14 @@ decl {\#include <FL/fl_ask.H>} {private global
 decl {\#include <string.h>} {private global
 }
 
+decl {void init_scheme(void);} {
```

Comment:
Just as an aside, you can actually put the `#include` statement into the **Code** section in the widget dialog. FLUID positions it correctly in the source for you, keeping things together. If you'd ever delete the widget, you would automatically delete the include statement as well.

Reply:
Thanks for the hint. I always have problems to insert such things like #include in the right place(s). Fluid is still a mystery to me (and I don't meant its code but its user interface). Admittedly I've never actively worked with fluid.

Reply:
FLUID is not really hard to use, but it does some hidden things that may be useful, but are unexpected. Working on it.